### ISSUE TITLE
Couldn't set up environment properly

### ISSUE THREAD

**Issue by Kentwhf** [AdrianNunez/Fall-Detection-with-CNNs-and-Optical-Flow#19]
Hi Andrian,

I read your paper and I am interested in trying it on my own dataset. However, I think the package versions are not compatible with each other. I tried running requirements.txt in terminal, but few issues came up. For example, I could only install Pypubsub 3.3.0 instead of 4.0.0, with python 2.7.17, and also tensorflow 1.12.0 not installable. I tried getting the version of 1.15.0, but it prompted that it only supports python 3.4 or higher. 

My OS: Windows 10 Version 10.0.18362 Build 18362

Is it possible for you to update the dependency list a bit? Is there a way of installing the modules with python 3.7?

Thanks in advance!

**Comment by AdrianNunez** [comment]
Hi,

If I remember correctly, I believe Pypubsub is not necessary (I will erase it from the file in that case). For Tensorflow, try other updated versions for python 2.7. I believe you can install older versions from sources; in fact, you may be able to find Docker containers with those versions already installed.

**Comment by Kentwhf** [comment]
Thanks Adrain.

I have already tried installing old versions of TensorFlow using pip and conda, but again there come some conflicts between packages. I think official TensorFlow only supports python 3.5 or higher on windows now. Eventually, I was able to install tensorflow-1.10.0-cp27-cp27m-win_amd64 in local. 

Another problem I ran into was pycocotools. Although I don't know if it is necessary, I spent a few hours figuring out how to install it. I found a solution for Python2.7 in https://github.com/cocodataset/cocoapi/issues/51.
